### ISSUE TITLE
Dynamic Table Column showcase produce empty rows

### ISSUE THREAD

**Issue by rinkesj** [primefaces/primefaces-showcase#353]
Dynamic Table Column showcase produce empty rows when it is used from showcase.
https://www.primefaces.org/showcase/ui/data/datatable/columns.xhtml?jfwid=f1768

XHTML definition for columns is not complete and that produce correct columns but empty rows.

actual column definition missing outputText:
```
<p:columns value="#{dtColumnsView.columns}" var="column" headerText="#{column.header}"
                       field="#{column.property}"/>
```

workin columns definition:
```
<p:columns value="#{dtColumnsView.columns}" var="column"
	headerText="#{column.header}" field="#{column.property}">
	<h:outputText value="#{product[column.property]}"></h:outputText>
</p:columns>
```

**Comment by melloware** [comment]
Hmm let me look it should not need the OutputText.

**Comment by melloware** [comment]
I don't understand what you mean.  I don't see any empty rows in the Showcase?

**Comment by rinkesj** [comment]
> I don't understand what you mean. I don't see any empty rows in the Showcase?

I copied dynamic column example code from showcase into base maven jsf project and it print always empty column.
![image](https://user-images.githubusercontent.com/1951770/115616595-a93b0400-a2f0-11eb-9886-89ba974f2325.png)

But you are right, It is because primefaces 8.0. I just notice it and when I used version 10.0.0 it works. 

Sorry for false alarm.
